dice/builtin_commands.go

```diff
@@ -1718,19 +1718,20 @@ func (d *Dice) registerCoreCommands() {
 		".pc tag <角色名> | <角色序号> // 当前群绑卡/解除绑卡(不填角色名)\n" +
 		".pc untagAll <角色名> | <角色序号> // 全部群解绑\n" +
 		".pc list // 列出当前角色和序号\n" +
-		".pc rename <角色名> <新角色名>\n" +
+		".pc rename <新角色名> // 将当前绑定角色改名\n" +
+		".pc rename <角色名|序号> <新角色名> // 将指定角色改名 \n" +
 		// ".ch group // 列出各群当前绑卡\n" +
 		".pc save <角色名> // [不绑卡]保存角色，角色名可省略\n" +
 		".pc load <角色名> | <角色序号> // [不绑卡]加载角色\n" +
 		".pc del/rm <角色名> | <角色序号> // 删除角色 角色序号可用pc list查询\n" +
```

Comment:
暮星改过这里的格式，需要同步一下master分支

```suggestion
		".pc load (<角色名> | <角色序号>) // [不绑卡]加载角色\n" +
		".pc del/rm (<角色名> | <角色序号>) // 删除角色 角色序号可用pc list查询\n" +
```

Reply:
同步了